sass/_TextEllipsisTooltip.scss

```diff
@@ -0,0 +1,34 @@
+@import 'Variables';
+
+.coveo-text-ellipsis-tooltip {
```

Comment:
I'm wondering how that tooltip gets interpreted by a screen reader ?

Would a blind user still be able to know the "full" facet value, without this visual tooltip ?

Reply:
yes because the value is "cut" visually with CSS. It's entirely there in the dom